include/branch.rkt

```diff
@@ -2,11 +2,10 @@
 
 (require ffi/unsafe
          ffi/unsafe/alloc
-         "define.rkt"
          "types.rkt"
          "buffer.rkt"
          "refs.rkt"
-         "utils.rkt")
+         libgit2/private)
```

Comment:
do you want to mix path-style and lib-style refs here? wouldn't it make more sense to use "private.rkt"? Among other things, it makes it impossible to test the code without wiring it in with a 'raco pkg install'

Reply:
I don't feel strongly about this, but I tend to use lib-style for anything that involves going up the directory hierarchy. But I wouldn't mind changing it, though testing without `raco pkg install` would be require getting the native library, anyway.

Reply:
At some point in the last 3 years I changed my mind about this and went back to e.g. `"../private/base.rkt"`.